### PR TITLE
Add props to SkeletonTable

### PR DESCRIPTION
**Pull request by HenryQuillin**
Allows SkeletonTable to be more customizable for future use. Also, fixes overflow issue on mobile viewports.

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-36mmd7q Loading skeletons](https://app.clickup.com/t/36mmd7q)

File: src/pages/home/owners/Owners.tsx [hunk 1]
```diff
@@ -42,7 +45,7 @@ const Owners: FunctionComponent = () => {
             query={ query }
             onSearch={ handleSearch }
           />
-          <TableSkeleton />
+          <TableSkeleton cols={ viewportWidth && viewportWidth > 450 ? 3 : 2 } />
```
Comment (apolloabds) [inline src/pages/home/owners/Owners.tsx:48]: if these are the breakpoints:
```
    values: {
      xs: 0,
      sm: 600,
      md: 900,
      lg: 1060,
      xl: 1536,
    },
```
and on `src/pages/home/owners/OwnersTable.tsx` line 85: `display: { xs: "none", sm: "inline" },`

seems like we want the 450 value to be 600?, also we probably want to use `theme.breakpoints.values.sm` here 
```suggestion
          <TableSkeleton cols={ viewportWidth && viewportWidth > theme.breakpoints.values.sm ? 3 : 2 } />
```

If the change is made here, we should make it in `Library.tsx` as well, let me know your thoughts

Comment (escobarjonatan) [inline src/pages/home/owners/Owners.tsx:48]: if these are the breakpoints:
```
    values: {
      xs: 0,
      sm: 600,
      md: 900,
      lg: 1060,
      xl: 1536,
    },
```
and on `src/pages/home/owners/OwnersTable.tsx` line 85: `display: { xs: "none", sm: "inline" },`

seems like we want the 450 value to be 600?, also we probably want to use `theme.breakpoints.values.sm` here 
```suggestion
          <TableSkeleton cols={ viewportWidth && viewportWidth > theme.breakpoints.values.sm ? 3 : 2 } />
```

If the change is made here, we should make it in `Library.tsx` as well, let me know your thoughts

**Review by escobarjonatan — APPROVED** [review]

**Comment by escobarjonatan** [comment]
Closing this PR as i tried to merge but there were issues with the build "Error: Resource not accessible by integration" Applied your changes and pushed them up to my branch. It seems to be fine, not sure why 🤷🏽‍♂️. Thanks for putting this PR together 🙏🏽

**Pull request closed without merging**